sql/src/main/java/io/crate/operation/projectors/writer/OutputFile.java

```diff
@@ -46,7 +45,7 @@ public OutputFile(URI uri, Settings settings) {
     }
 
     @Override
-    public void open() throws IOException {
+    public OutputStream acquireOutputStream() throws IOException {
```

Comment:
added it: https://github.com/crate/crate/pull/2556/files#diff-55e6dd88d485c910b9b88c2ffaf10183R32